rest-service/manager_rest/resource_manager.py

```diff
@@ -279,6 +280,8 @@ def install_plugin(self, plugin):
             self._validate_plugin_yaml(plugin)
 
         if not utils.plugin_installable_on_current_platform(plugin):
+            remove_status_prefix(plugin)
```

Comment:
Remove from this PR and create a different PR for it as discussed.
For the different PR, a plugin should be deleted from the DB if it cannot be installed.

Reply:
OK After some further discussions, this solution should suffice, no need to delete from the DB.